src/apps/jsonld_converter/service/document/activity.py

```diff
@@ -492,10 +492,12 @@ def _export_subscales(self) -> SubscaleSetting | None:
                             continue
                         if item.name in subscale_names:
                             item.type = SubscaleItemType.SUBSCALE
-                        else:
-                            raise SubscaleParsingError(
-                                f'Item name "{item.name}" not found'
-                            )
+                        elif subscale.items is not None:
```

Comment:
looks weird. It's done in cycle by those items, how it can be null?
For what it is done in general?

Reply:
actually, can be just `else:`.  
overall, if item name is not found in items list, just removing, instead of raising Exception

Reply:
that's my fix, mypy complained